integration/setup/setup.go

```diff
@@ -132,14 +135,14 @@ func installAWSOperator(config Config) error {
 
 	}
 
-	err = config.Host.InstallBranchOperator("aws-operator", "awsconfig", values)
+	err = config.Release.InstallOperator(ctx, "aws-operator", release.NewStableVersion(), values, providerv1alpha1.NewAWSConfigCRD())
```

Comment:
This here replaces `InstallBranchOperator` and should use `release.NewVersion` instead of `release.NewStableVersion`. Just found it because I tried to understand how it should work in Azure.

Reply:
Should probably be `release.NewVersion(env.CircleSHA())`.

Reply:
Yes correct. Thank you. I have more problems than this one here, because this thing installs encrypter secret in a wrong namespace. I leave that for tomorrow.